docs/getting-started/sensor-full-disk-access.asciidoc

```diff
@@ -3,23 +3,91 @@
 
 Elastic Endpoint Security requires full disk access to protect you from malware and other cybersecurity threats. Full Disk Access permissions is a new privacy feature introduced in macOS Mojave (10.14) that prevents some applications from accessing your data. This means you need to manually grant permission for Elastic Endpoint Security to access these protected areas of your Mac.
 
-This article describes how to enable full disk access for both the Elastic Agent, required in order to enable Elastic Endpoint Security, and the legacy Endgame sensor.
+This article describes how to enable full disk access for the required security system extensions, the Elastic Endpoint sensor, and the legacy Endgame sensor.
+
+[discrete]
+[[macos-permissions]]
+== macOS permissions
+
+The behavior of the Endgame Sensor and Elastic Endpoint differs based on your macOS version. MDM/JAMF users can pre-approve all full disk access without granting permission to the sensors. However, non-MDM/JAMF users will be prompted or not prompted to enable full disk access for required security files depending on the macOS version and sensor type:
+
+**Endgame Sensor**
+
+- `10.13, 10.14, 10.15`: Users cannot proceed with installation without first granting the sensor the ability to load a <<enable-kernel-extension,kernel extension>>. During installation, you will be prompted to go to **System preferences** and approve loading the kernel. Upon approval, installation proceeds. 
+
+- `11.0` (Big Sur): Users cannot proceed with installation without first granting the sensor the ability to load <<system-extension>>. During installation, you will be prompted to go to **System preferences** and approve loading the system extension. Upon approval, a second prompt appears to enable **Network Filtering**. Approve this final prompt for installation to proceed.
++
+You also must grant full disk access to `com.endgame.systemextension`
+
+- `10.14.6+, 10.15, 11.0`: Grant the <<endpoint-endgame-sensor,esensor>> full disk access.
+
+
+**Elastic Endpoint**
+
+No prompts to approve the <<enable-kernel-extension,kernel extension>>, <<system-extension>>, or the <<endpoint-endgame-sensor,elastic-endgame>>, due to installation happening through the {agent}. After installation, Endpoint policies **will fail** to detect events until you approve and enable kernel or system extension loading and full disk access for each version, as reflected in the **Administration > Endpoints** page of the security application.
```

Comment:
I feel like these bullet points can be a little confusing w/ just the links in them. What about adding `"Approve the kernel extension", etc. as the link text?